{"name": "test_login[data1]", "status": "passed", "attachments": [{"name": "log", "source": "7c3123c0-ef2a-4a83-bef2-fc24d1583dcf-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'case_id': 'login_case_002', 'case_title': '用户输入错误的用户名、\\n正确的密码，点击登录', 'test_data': {'username': 'alive1', 'password': 123456}, 'exp': {'by': 'xpath', 'locater': \"//*[@id='common-prompt']/div/p\", 'exp_value': '登录帐号不存在'}}"}], "start": 1663419356884, "stop": 1663419364421, "uuid": "74238e93-5b24-4f0a-957e-3a2c1ad6ae09", "historyId": "6a42ab545080d6017bf21853c464e0cc", "testCaseId": "c6db3f0583de5245fe1a62b1d735a86c", "fullName": "run.TestLogin#test_login", "labels": [{"name": "suite", "value": "run"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-5FKMFO0H"}, {"name": "thread", "value": "4888-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "run"}]}